crates/rpc/rpc-builder/src/auth.rs

```diff
@@ -24,6 +24,8 @@ use std::{
     net::{IpAddr, Ipv4Addr, SocketAddr},
     time::{Duration, SystemTime, UNIX_EPOCH},
 };
+/// Default max response size in MB.
+pub(crate) const RPC_DEFAULT_MAX_RESPONSE_SIZE_MB: u32 = 100;
```

Comment:
I don't want an additional variable for this,

I'd rather have another field for the authconfig type instead that we then set in cli args

Reply:
Sounds good i made the requested changes